packages/wrangler/src/pages/dev.tsx

```diff
@@ -379,15 +379,16 @@ export const Handler = async ({
 					plugins: [
 						esbuildAliasExternalPlugin({
 							__ENTRY_POINT__: entrypointFile,
+							"./pages-dev-util": resolve(
```

Comment:
Yeah, nice, this is what I was talking about when I meant it might have trouble locating the relative file.